tools/fix_version_numbers.py

```
@@ -6,11 +6,12 @@
 
 
 class Version:
-    def __init__(self, major: int, minor: int, patch: int):
+    def __init__(self, major: int, minor: int, patch: int, year: Optional[int] = None):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)